ng-dialog/ng-dialog.d.ts

```diff
@@ -41,18 +52,54 @@ declare module angular.dialog {
 		 * @returns {}
 		 */
 		setDefaults(defaultOptions: IDialogOptions): void;
+
+		/**
+		 * Adds an additional listener on every $locationChangeSuccess event and gets update version of html into dialog.
+		 * May be useful in some rare cases when you're dependant on DOM changes, defaults to false.
+		 * @param {boolean} force
+		 */
+		setForceHtmlReload(force: boolean) : void;
+
+		/**
+		 * Adds additional listener on every $locationChangeSuccess event and gets updated version of body into dialog.
+		 * Maybe useful in some rare cases when you're dependant on DOM changes, defaults to false. Use it in module's
+		 * config as provider instance:
+		 * @param {boolean} force
+		 */
+		setForceBodyReload(force: boolean) : void;
 	}
 
 	/**
 	 * Dialog Scope which extends the $scope.
 	 */
-	interface IDialogScope extends angular.IScope {
+	interface IDialogOpenScope extends angular.IScope {
 		/**
 		 * This allows you to close dialog straight from handler in a popup element.
 		 * @param value Any value passed to this function will be attached to the object which resolves on the close promise for this dialog.
 		 * For dialogs opened with the openConfirm() method the value is used as the reject reason.
 		 */
 		closeThisDialog(value?: any): void;
+
+		/**
+		 * Any serializable data that you want to be stored in the controller's dialog scope.
+		 * From version 0.3.6 $scope.ngDialogData keeps references to the objects instead of copying them.
+		 */
+		ngDialogData : {};
```

Comment:
I think this should be as `string | {} | any[]; `, to match the `data` provided within the `IDialogOpenOptions`